os/arch/arm/src/amebasmart/Kconfig

```diff
@@ -17,6 +17,7 @@ config ARCH_CHIP_RTL8730E
 	select ARMV7A_HAVE_GICv2
 	select ARCH_HAVE_FPU
 	select AMEBASMART_WIFI
+	select ARCH_BOARD_HAVE_SECOND_FLASH
```

Comment:
It should be valid with HW Rev 5. Is it possible to give any conditional on this? @ewoodev Could you check this together?

Reply:
https://github.com/Samsung/TizenRT/blob/cf41d8ad8333a95dc375331578c7d5920d4acc9d/os/board/rtl8730e/Kconfig#L39-L44

Please add condition if RTL8730E_BOARD_REVISION > 5